Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -1692,6 +1692,16 @@ - (BOOL)_isWindowResizingEnabled
 
 #endif // HAVE(UIKIT_RESIZABLE_WINDOWS)
 
+#if ENABLE(INTERACTION_REGIONS_IN_EVENT_REGION)
+- (void)_setInteractionRegionsEnabled:(BOOL)enabled
+{
+    WKWEBVIEW_RELEASE_LOG("%p -[WKWebView _setInteractionRegionsEnabled] page - %p", self, _page.get());
+    if (_page && _page->preferences().interactionRegionsEnabled())
```

Comment:
@hortont424 are you ok with this change if it doesn't allow turning on interaction regions when its disabled in the WKPreferences?

So it can get disabled on the WebPage instance but not enabled if interaction regions are disabled in WKPreferences

Reply:
that logic makes sense to me